packages/react/src/components/Search/Search.js

```diff
@@ -5,283 +5,251 @@
-     */
-    closeButtonLabelText: PropTypes.string,
-
-    /**
-     * Optionally provide the default value of the `<input>`
-     */
-    defaultValue: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-
-    /**
-     * Specify whether the `<input>` should be disabled
-     */
-    disabled: PropTypes.bool,
-
-    /**
-     * Specify a custom `id` for the input
-     */
-    id: PropTypes.string,
-
-    /**
-     * Provide the label text for the Search icon
-     */
-    labelText: PropTypes.node.isRequired,
-
-    /**
-     * Specify light version or default version of this control
-     */
-    light: PropTypes.bool,
-
-    /**
-     * Optional callback called when the search value changes.
-     */
-    onChange: PropTypes.func,
-
-    /**
-     * Optional callback called when the search value is cleared.
-     */
-    onClear: PropTypes.func,
-
-    /**
-     * Provide a handler that is invoked on the key down event for the input
-     */
-    onKeyDown: PropTypes.func,
-
-    /**
-     * Deprecated in favor of `placeholder`
-     */
-    placeHolderText: deprecate(
-      PropTypes.string,
-      `\nThe prop \`placeHolderText\` for Search has been deprecated in favor of \`placeholder\`. Please use \`placeholder\` instead.`
-    ),
-
-    /**
-     * Provide an optional placeholder text for the Search.
-     * Note: if the label and placeholder differ,
-     * VoiceOver on Mac will read both
-     */
-    placeholder: PropTypes.string,
-
-    /**
-     * Rendered icon for the Search.
-     * Can be a React component class
-     */
-    renderIcon: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
-
-    /**
-     * Specify the search size
-     */
-    size: PropTypes.oneOf(['sm', 'md', 'lg', 'xl']),
-
-    /**
-     * Specify whether the Search should be a small variant
-     */
-
-    /**
-     * Specify whether the load was successful
-     */
-    small: deprecate(
-      PropTypes.bool,
-      `\nThe prop \`small\` for Search has been deprecated in favor of \`size\`. Please use \`size="sm"\` instead.`
-    ),
-
-    /**
-     * Optional prop to specify the type of the `<input>`
-     */
-    type: PropTypes.string,
 
-    /**
-     * Specify the value of the `<input>`
-     */
-    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-  };
-
-  static contextType = FeatureFlagContext;
-
-  static defaultProps = {
-    type: 'text',
-    placeholder: '',
-    closeButtonLabelText: 'Clear search input',
-    onChange: () => {},
-    onClear: () => {},
-  };
-
-  state = {
-    hasContent: this.props.value || this.props.defaultValue || false,
-    prevValue: this.props.value,
-  };
-
-  static getDerivedStateFromProps({ value }, state) {
-    const { prevValue } = state;
-    return prevValue === value
-      ? null
-      : {
-          hasContent: !!value,
-          prevValue: value,
-        };
+const Search = React.forwardRef(function Search(
```

Comment:
IIRC due to the forwardRef there isn't a name associated with this component. Adding a `displayName` to this might fix some of the tests that are searching for the name "Search"

Reply:
https://github.com/carbon-design-system/carbon/pull/9591/

Reply:
That worked!